packages/slate/test/index.test.js

```diff
@@ -2,25 +2,25 @@
  * Dependencies.
  */
 
-import { resetKeyGenerator } from '..'
+import { resetKeyGenerator } from '../src/'
 
 /**
  * Tests.
  */
 
 describe('slate', () => {
+  /**
+   * Reset Slate's internal key generator state before each text.
+   */
+
+  beforeEach(() => {
+    resetKeyGenerator()
+  })
```

Comment:
Can you remove the comment here, and simplify to:

```js
beforeEach(resetKeyGenerator)
```

Reply:
Fixed~